src/ElectricPokemon.java

```diff
@@ -0,0 +1,29 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon  extends Pokemon{
+
+    String[] attacks = {"thunderPunch" ,"electroBall" ,"thunder" ,"voltTackle"};
```

Comment:
De meeste mensen maken hier gelijk een List, maar een array in combinatie met `asList` in de getter is ook een prima oplossing.